e2e/tests/upgrades/upgrade_test.go

```diff
@@ -74,22 +74,13 @@ func (s *UpgradeTestSuite) UpgradeChain(ctx context.Context, chain *cosmos.Cosmo
 		Info:   fmt.Sprintf("upgrade version test from %s to %s", currentVersion, upgradeVersion),
 	}
 
-	if testvalues.GovV1MessagesFeatureReleases.IsSupported(chain.Config().Images[0].Version) {
```

Comment:
Is this just not needed anymore, or will this break compatibility tests in any way we need to deal with later?

Reply:
Pretty sure it should be fine. As 0.47 (ibc-go/v7) has govv1 msg execution.